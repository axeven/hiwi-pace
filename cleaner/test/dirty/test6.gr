p tw 10 10
1 2
1 4
2 3
3 4
4 5
5 9
6 7
6 8
7 8
9 10
