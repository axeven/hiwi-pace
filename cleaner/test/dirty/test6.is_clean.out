test6.gr dirty 10 10 not connected 0 1 8 1
